src/components/tabs/tabs.test.tsx

```diff
@@ -861,6 +862,78 @@ test("when errors and warnings are both present in a tab, only the error icon is
   ).not.toBeInTheDocument();
 });
 
+test("error and warning icons are displayed corectly when the new validation flag is present and the tabs have position top", async () => {
+  const user = userEvent.setup();
+  render(
+    <CarbonProvider validationRedesignOptIn>
+      <Tabs selectedTabId="tab-1">
+        <Tab tabId="tab-1" title="Tab 1" errorMessage="error">
+          Content for tab 1
+          <Textbox error onChange={() => {}} />
+        </Tab>
+        <Tab tabId="tab-2" title="Tab 2" warningMessage="warning">
+          Content for tab 2
+          <Textbox warning onChange={() => {}} />
+        </Tab>
+      </Tabs>
+    </CarbonProvider>
+  );
+
+  expect(
+    within(screen.getByRole("tab", { name: "Tab 1" })).getByTestId("icon-error")
+  ).toBeInTheDocument();
+
+  expect(
+    within(screen.getByRole("tab", { name: "Tab 2" })).getByTestId(
+      "icon-warning"
+    )
+  ).toBeInTheDocument();
+
+  await user.click(screen.getByRole("tab", { name: "Tab 1" }));
+  expect(screen.getByText("Content for tab 1")).toBeVisible();
+
+  await user.click(screen.getByRole("tab", { name: "Tab 2" }));
+  expect(screen.getByText("Content for tab 2")).toBeVisible();
+
+  expect(true).toBe(true);
```

Comment:
**nit-pick:** Assuming this was for testing and should've been removed?

Reply:
Removed the testing assertion.